tests/setups/virtual_drive.py

```diff
@@ -1,6 +1,5 @@
-from tests import dictionaries
+from ingenialink.ethernet.network import VIRTUAL_DRIVE_DICTIONARY
+
 from tests.setups.descriptors import VirtualDriveSetup
 
-TESTS_SETUP = VirtualDriveSetup(
-    ip="127.0.0.1", dictionary=dictionaries.VIRTUAL_DRIVE_XDF_PATH, port=1061
-)
+TESTS_SETUP = VirtualDriveSetup(ip="127.0.0.1", dictionary=VIRTUAL_DRIVE_DICTIONARY, port=1061)
```

Comment:
Why this change? If we want to separate virtual drive from ingenialink shouldn't this make it harder?